### DISCUSSION TITLE
blog/faq-on-hate-china-party

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#43] [Announcements]
# blog/faq-on-hate-china-party

Two thousand seven hundred and fifty words

https://pathos.page/blog/faq-on-hate-china-party

**Comment by zmk-ink** [comment]
我年轻时就是倔强的恨国党，现在想起来完全是对这个世界的片面认知，是把自我当作真理的愚蠢。世界是五彩缤纷的，你没有办法对一个没吃过原味夏威夷果的人描述它的味道。所以是的，别浪费口舌，别让他们影响到自己的心情和生活。

**Reply by Pathsis** [reply]
哈哈，谁小时候没叛逆过？但如果长大了，知识和阅历增长了，对历史和现实也有更多认识和观察了，反而变成了一根筋、二极管，就真得反思下了。

**Reply by seeyousuperman** [reply]
十分认同二位

**Comment by zmk-ink** [comment]
刚刚看到一段话，总结得很好：

When you sell out your own country and insult your leaders, the US-led West die-hard free speech defenders will applaud you, make you a hero, and even give you a Nobel Peace Prize.

But suddenly, free speech is no more when you do the same to them.

**Reply by jungweipai** [reply]
of course, as long as there are people, there is no absolute objectivity, for always
